app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -71,60 +39,68 @@ To be eligible for a bursary, you’ll need at least one of the following:
 
 If your course combines 2 subjects (for example, English and history), at least 50% of your course content must be made up of the bursary subject to be eligible.
 
-You’ll also need to meet the bursary scheme’s individual terms and conditions.
+#### How bursaries are paid
 
-[Find out more about your eligibility for teaching bursaries](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2024-to-2025).
+A bursary is paid over a minimum of 10 months by your teacher training provider. They will confirm when you will get the payments. 
```

Comment:
Previously we said `over the duration of your course`. I know that we can't be really definitive as this depends on the provider but we are now not giving any indication of when the payments will be? Worth asking policy if we can be more detailed?

As it is written currently, I might not be certain that I'd be paid during my course - could be after!